winui/Scheduler/Month-View.md

```diff
@@ -260,6 +260,24 @@ this.Schedule.MonthViewSettings.TrailingDaysVisibility = Visibility.Collapsed;
 
 ![customize-leading-and-trailing-days-visibility-in-winui-scheduler-month-view](Month-View_Imges/customize-leading-and-trailing-days-visibility-in-winui-scheduler.png)
 
+## Customizing the Number of Visible Weeks
+The number of week visible in the month view can be changed by setting the NumberOfVisibleWeeks property of the MonthViewSettings in SfScheduler. The default value is 6, and the valid range is from 1 to 6. Even if leading and trailing days are hidden, they will appear when changing the number of visible weeks, as no additional days exist during this change.
```

Comment:
The number of week visible in the month view can be changed by setting the `NumberOfVisibleWeeks` property of `MonthViewSettings` in `SfScheduler`.  By default, the number of visible weeks is 6.